contextily/plotting.py

```diff
@@ -6,6 +6,7 @@
 from .tile import _calculate_zoom, bounds2img, _sm2ll, warp_tiles, _warper
 from rasterio.enums import Resampling
 from rasterio.warp import transform_bounds
+from shapely.geometry import Polygon
```

Comment:
I don't think this is necessary? The mask you build later is directly from geojson, no `shapely` required.

Reply:
good point!